Libraries/react-native/react-native.js

```diff
@@ -20,6 +20,7 @@
 var ReactNative = Object.assign(Object.create(require('React')), {
   // Components
   ActivityIndicatorIOS: require('ActivityIndicatorIOS'),
+  ART: require('ReactNativeART'),
```

Comment:
Ultimately, this will go into the `react-art` package and this will be removed, but since it depends on `react` we'll need to wait for #2985